tests/dummy/app/pods/demo/template.hbs

```diff
@@ -32,16 +32,17 @@
     yScale=(d3-linear-scale)
     as |chart|
   }}
-    {{#chart.x-axis label='Line' as |tick|}}
-      {{to-fixed tick 0}}
-    {{/chart.x-axis}}
+    {{#chart.x-axis-svg label='Line' alignment='top' offset=-6 as |tick|}}
+      <text x=-7 y=0>Foo {{to-fixed tick 0}}</text>
```

Comment:
I'd like to see an issue for the need to do `x=-7` as well, we should be able to auto-calculate that (in the future)